src/components/LandingPage/index.tsx

```diff
@@ -15,17 +17,17 @@ export function LandingPage() {
                 Trilhe o caminho da estabilidade financeira
               </h1>
               <div className="w-full flex justify-center  mt-4 md:mt-10">
-                <button className="py-2 px-4 md:w-64 md:items-self-center rounded-md bg-green-500 md:py-4  md:px-3">
-                  <a href="/api/auth/login" className="text-sm md:text-2xl ">
+                <Button className="py-2 px-4  hover:bg-green-300 md:w-64 md:items-self-center rounded-md bg-green-500 md:py-4  md:px-3">
+                  <Link href="/api/auth/login" className="text-sm md:text-2xl ">
```

Comment:
Não faz sentido um link dentro de um botão, e o botão também não esta especificado seu type nem a ação ao ser clicado.
Se for só por conta de estilização, aplique elas ao link e exclua o botão.
Se precisar de ser um botão, uma alternativa é na função de onclick do botão, executar o redirecionamento para a página que você quer.